<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Prasanna" failures="1" tests="28" name="features.TestRunner" time="146.325" errors="0" timestamp="2024-09-03T12:37:59 IST" skipped="0">
  <testcase classname="features.TestRunner" name="runScenario" time="8.033"/>
  <testcase classname="features.TestRunner" name="runScenario" time="7.033"/>
  <testcase classname="features.TestRunner" name="runScenario" time="0.034"/>
  <testcase classname="features.TestRunner" name="runScenario" time="3.175"/>
  <testcase classname="features.TestRunner" name="runScenario" time="3.441"/>
  <testcase classname="features.TestRunner" name="runScenario" time="5.660"/>
  <testcase classname="features.TestRunner" name="runScenario" time="4.397"/>
  <testcase classname="features.TestRunner" name="runScenario" time="8.066"/>
  <testcase classname="features.TestRunner" name="runScenario" time="8.587"/>
  <testcase classname="features.TestRunner" name="runScenario" time="5.428"/>
  <testcase classname="features.TestRunner" name="runScenario" time="4.603"/>
  <testcase classname="features.TestRunner" name="runScenario" time="3.334"/>
  <testcase classname="features.TestRunner" name="runScenario" time="4.819"/>
  <testcase classname="features.TestRunner" name="runScenario" time="5.725"/>
  <testcase classname="features.TestRunner" name="runScenario" time="4.292"/>
  <testcase classname="features.TestRunner" name="runScenario" time="3.445"/>
  <testcase classname="features.TestRunner" name="runScenario" time="7.303"/>
  <testcase classname="features.TestRunner" name="runScenario" time="7.441"/>
  <testcase classname="features.TestRunner" name="runScenario" time="4.694"/>
  <testcase classname="features.TestRunner" name="runScenario" time="0.921"/>
  <testcase classname="features.TestRunner" name="runScenario" time="4.563"/>
  <testcase classname="features.TestRunner" name="runScenario" time="5.610">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at stepDefinitions.LoginSteps.i_get_same_password_error(LoginSteps.java:347)
at âœ½.I get error for same password as previous(file:///C:/Users/prasa/eclipse-workspace/YaaayFreshers/src/test/java/features/Login.feature:166)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="features.TestRunner" name="runScenario" time="5.319"/>
  <testcase classname="features.TestRunner" name="runScenario" time="8.084"/>
  <testcase classname="features.TestRunner" name="runScenario" time="5.904"/>
  <testcase classname="features.TestRunner" name="runScenario" time="5.974"/>
  <testcase classname="features.TestRunner" name="runScenario" time="4.528"/>
  <testcase classname="features.TestRunner" name="runScenario" time="5.912"/>
</testsuite> <!-- features.TestRunner -->
